src/lib/pages/account-details/components/AccountTitle.tsx

```diff
@@ -3,24 +3,27 @@ import { Flex, Heading, Image, Skeleton } from "@chakra-ui/react";
 import { useMoveConfig } from "lib/app-provider";
 import { CustomIcon } from "lib/components/icon";
 import type { AccountData } from "lib/services/types";
-import { useInitiaUsernameByAddress } from "lib/services/username";
 import type { AccountLocalInfo } from "lib/stores/account";
-import type { HexAddr, Option } from "lib/types";
+import type { Option } from "lib/types";
+
+interface AccountTitleProps {
+  accountData: Option<AccountData>;
+  accountLocalInfo: Option<AccountLocalInfo>;
+  data?: {
+    username: string | null;
+  };
+  isLoading?: boolean;
+  isFetching?: boolean;
```

Comment:
```ts
interface AccountTitleProps {
  accountData: Option<AccountData>;
  accountLocalInfo: Option<AccountLocalInfo>;
  initiaUsernameData: Option<initiaUsernameDataResponse>
  isInitiaUsernameDataLoading: boolean;
  isInitiaUsernameDataFetching: boolean;
```

- Pls rename properties naming convention to make it more specific
- `isLoading` and `isFetching`, both are `boolean` no need to add `?`
- For `data` please use `Option` instead of `?`
- `initiaUsernameDataResponse` as comment below, can you create a interface for the initia username func returning type